scripts/index.js

```diff
@@ -92,4 +92,28 @@ function whosLastName(nameStr){
     return results;
 }
 
-// console.log(whosLastName("Targaryen"))
\ No newline at end of file
+// console.log(whosLastName("Targaryen"))
+
+function histogramFun(){
+    let results = {};
+    let result = '';
+    characters.forEach(function(charObj){
+        charObj.allegiances.forEach(function(alStr){
+            for (let i = (alStr.length - 1); i > alStr.length - 6; i--){
+                if ("1234567890".includes(alStr[i])){
+                    result = alStr[i] + result;
```

Comment:
oh ho ho! I get it. You're nabbing the house number!
Clever, but consider adding some comments explaining your approach.